functions/groups.js

```diff
@@ -492,17 +492,7 @@ const CreateGroups = {
     })
     var activity = null
     if (matchingActivities.length === 0) {
-      activity = {
-        id: ++maxActivityId,
-        activityCode: round.id(),
-        childActivities: [],
-        scrambleSetId: null,
-        extensions: [],
-        startTime: start.toISO(),
-        endTime: end.toISO(),
-        name: round.id()
-      }
-      matchingRooms[0].activities.push(activity)
+      return ['Could not find matching activity for times']
```

Comment:
This one is likely the part I'm most unsure about; I did mess up a couple of times when copy/paste some times, which lead to creating unintended "parent" activities in the schedule.
I'm not sure if you had a legit use case for the original behavior, since I'd guess you would be creating groups for already existing rounds?

Reply:
This part has never been actually used (I used a GUI schedule editor last year, and wrote CreateGroups after Nats23). So I will definitely defer to your experience if there are usability issues!